docs/resources/rfs_private_hook.md

```diff
@@ -0,0 +1,166 @@
+     dedicated regions refer to dedicated regions that only carry the same type of business or provide business services
+     to specific tenants).
+
+* `policy_body` - (Optional, String) Specifies the policy content of the private hook.  
+  Only policy templates written in [Rego](https://www.openpolicyagent.org/docs/latest/policy-language) language that are
+  recognized by the OPA open source engine are supported.
+
+-> Exactly one of the `policy_uri` and `policy_body` must be set.
+
+* `configuration` - (Optional, List) Specifies the configuration of the private hook, that can specify the target
+  resource stack where the private hook takes effect and the behavior of the resource stack after the private hook
+  verification fails.  
+  The [configuration](#private_hook_configuration) structure is documented below.
+
+* `keep_old_version` - (Optional, Bool) Specifies whether keeping old version while updating hook version.  
+  Defaults to **false**.
+
+  -> A maximum of `199` historical versions can be created for a hook name.
+
+<a name="private_hook_configuration"></a>
+The `configuration` block supports:
+
+* `target_stacks` - (Optional, String) Specifies the target resource stack for the private hook to take effect.  
+  The valid values are as follows:
+  + **NONE**: This private hook will not be applied to any resource stack.
+  + **ALL**: This private hook will be applied to all resource stacks under the account.
+
+* `failure_mode` - (Optional, String) Specifies the behavior after private hook verification fails.  
+  The valid values are as follows:
+  + **FAIL**: The resource stack will stop deploying after this private hook verification fails, and the resource stack
+    status will be updated to DEPLOYMENT_FAILED.
+  + **WARN**: After this private hook verification fails, only a warning message will be displayed through the resource
+    stack event, but it will not affect the resource stack deployment.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `created_at` - The creation of the private hook, in RFC3339 format.
+
+* `updated_at` - The latest update of the private hook, in RFC3339 format.
+
+## Import
+
+Private hooks can be imported using their `name`, e.g.
+
+```bash
+$ terraform import huaweicloud_rfs_private_hook.test <name>
+```
+
+Note that the imported state may not be identical to your resource definition, due to some attributes missing from the
+API response. The missing attributes include: `policy_uri`, `policy_body` and `keep_old_version`. It is generally
+recommended running `terraform plan` after importing a hook. You can keep the resource the same with its definition bo
+choosing any of them to update. Also you can ignore changes as below.
+
+```hcl
+resource "huaweicloud_rfs_private_hook" "test" {
+  ...
+
+  lifecycle {
+    ignore_changes = [
+      policy_uri,
+      keep_old_version,
+    ]
```

Comment:
```suggestion
    policy_body]
```

Reply:
only one argument can be configured at the same time